tests/conftest.py

```diff
@@ -0,0 +1,116 @@
+# conftest.py
+# See: https://docs.pytest.org/en/7.1.x/reference/fixtures.html#conftest-py-sharing-fixtures-across-multiple-files
+"""
+The conftest.py file serves as a means of providing fixtures for an
+entire directory. Fixtures defined in a conftest.py can be used by any
+test in that package without needing to import them (pytest will
+automatically discover them).
+"""
+
+import logging
+import os
+import socket
+from typing import Any, AsyncGenerator, cast, Dict
+
+from pymongo import MongoClient
+from pymongo.errors import ServerSelectionTimeoutError
+import pytest
+import pytest_asyncio
+from rest_tools.client import RestClient
+
+from file_catalog.mongo import Mongo
+from file_catalog.server import create
+
+logger = logging.getLogger(__name__)
+
+MonkeyPatch = pytest.MonkeyPatch
```

Comment:
Seems a bit odd this isn't part of an `import ... as ...` statement, but it works

Reply:
Yeah, some time ago we used LGTM as a code quality scan.

For LTA it identified doing things like:
`import pytest`
`from pytest import MonkeyPatch`

The recommended workaround was just this; `MonkeyPatch = pytest.MonkeyPatch`

Personally, I like it better the import way, so I'll change it. Sorry, LGTM.

Reply:
> The recommended workaround was just this